rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/processes/execution-advice/expectations.json

```diff
@@ -2,18 +2,18 @@
+{"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex57-execution-advice-trade-full-termination_correction-C12-20.xml","excludedPaths":0,"externalPaths":55,"outstandingMappings":22,"validationFailures":7,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex58-execution-advice-trade-initiation-F01-00.xml","excludedPaths":0,"externalPaths":62,"outstandingMappings":7,"validationFailures":5,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex59-execution-advice-trade-amendment-F02-00.xml","excludedPaths":0,"externalPaths":78,"outstandingMappings":19,"validationFailures":6,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex60-execution-advice-trade-amendment-correction-F02-10.xml","excludedPaths":0,"externalPaths":78,"outstandingMappings":19,"validationFailures":6,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex63-execution-advice-trade-initiation.xml","excludedPaths":0,"externalPaths":101,"outstandingMappings":7,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_FixedFloat","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex64-execution-advice-trade-initiation-correction.xml","excludedPaths":0,"externalPaths":104,"outstandingMappings":7,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_FixedFloat","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/processes/execution-advice/msg-ex69-commodity-swap-coal-physical-leg.xml","excludedPaths":0,"externalPaths":76,"outstandingMappings":28,"validationFailures":13,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Forward","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
Commodity forwards have an increase of 2 validation failures:
- If settlementPayout underlier Asset exists, then a price must exist
- SettlementPayout settlement terms is a mandatory attribute